docs/data/base/guides/next-js-app-router/next-js-app-router.md

```diff
@@ -0,0 +1,97 @@
+# Next.js App Router
+
+<p class="description">Learn how to use Base UI with the Next.js App Router.</p>
+
+:::info
+Starting fresh on a new App Router-based project?
+
+Jump right into the code with this [example repo](https://github.com/mui/material-ui/blob/master/examples/base-next-app-router-ts).
+:::
+
+## Next.js and React Server Components
+
+The Next.js App Router implements React Server Components, a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) introduced in React 18.
+
+To support the App Router, currently all components and hooks from Base UI and other MUI libraries are exported with the `"use client"` directive.
+
+:::warning
+React Server Components does not replace, and is separate from server-side rendering (SSR). Client Components are still server-rendered to HTML.
+
+For more details, see this [explanation](https://github.com/reactwg/server-components/discussions/4) by the React team.
+:::
+
+## Setting up Base UI with the App Router
+
+Base UI is styling-agnostic; setting up a Next.js App Router project largely depends on the styling solution:
+
+### Emotion
+
+If you're using Emotion, or something Emotion-based like MUI System, you can follow the [same steps](/material-ui/guides/next-js-app-router/#using-material-ui-with-a-custom-theme) outlined for Material UI.
```

Comment:
Maybe we should provide those instructions here in this doc too? Directing readers to the Material UI docs could still lead to confusion. For example, when I visit the page linked, it tells me to import a Material UI component (`ThemeProvider`), but I don't know if that component is also provided by Base UI, or if I have to install Material UI to keep going.

Reply:
OK – originally I just linked it because I didn't want CSS-in-JS to take up so much space 😓 

I'll simplify the Emotion setup by a little bit and move Tailwind up to be the first section